thirdparty/flatbuffers/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@
 set(FLATBUFFERS_BUILD_TESTS OFF CACHE BOOL "" FORCE)
 set(FLATBUFFERS_INSTALL OFF CACHE BOOL "" FORCE)
 set(FLATBUFFERS_CPP_STD ${CMAKE_CXX_STANDARD})
+set(CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO})
```

Comment:
OpenVINO does not use flatbuffers source code in its binaries, we use only `flatc` compiler and header files:
https://github.com/openvinotoolkit/openvino/blob/fcb9c9d5d6d9fafde6edc399ae7d79c1d4767ede/cmake/developer_package/frontends/frontends.cmake#L277-L279
So, LTO is not needed here.

Reply:
OK, I removed it.